starter-code/javascript/main.js

```diff
@@ -7,54 +8,78 @@ var secDec      = document.getElementById('secDec');
 var secUni      = document.getElementById('secUni');
 var milDec      = document.getElementById('milDec');
 var milUni      = document.getElementById('milUni');
+var splits      = document.getElementById('splits');
 
+function showTime() {   // display time on the chronometer
+    setInterval(printTime, 100)
+}
 
 function printTime() {
-
+   printMinutes();
+   printSeconds();
 }
 
 function printMinutes() {
-
+    var minuteString  = chrono.twoDigitsNumber(chrono.minutes);
+    minDec.innerHTML = parseInt(minuteString.charAt(0), 10);
+    minUni.innerHTML = parseInt(minuteString.charAt(1), 10)
 }
 
 function printSeconds() {
-
-}
-
-function printMilliseconds() {
-
+  var secondString = chrono.twoDigitsNumber(chrono.seconds);
+  secDec.innerHTML = parseInt(secondString.charAt(0), 10);
+  secUni.innerHTML = parseInt(secondString.charAt(1), 10);
+  return chrono.seconds
 }
+    
 
-function printSplit() {
-
-}
 
-function clearSplits() {
+/*** START AND STOP BUTTON */
 
-}
+btnLeft.addEventListener('click', function () {
 
-function setStopBtn() {
+ if (btnLeft.innerHTML == "START") {   // buttons on load: START and RESET  
+    chrono.startClick();
+     showTime();
+     btnLeft.innerHTML = "STOP";     // change to "STOP" and "SPLIT" 
+     btnRight.innerHTML = "SPLIT";  
+     btnRight.style.backgroundColor = "#0851ab"; 
+     btnLeft.style.backgroundColor = "#f14949"; 
+
+ } else  {
+     chrono.stopClick();           // change to "START" and "RESET" 
+     btnLeft.innerHTML = "START";
+     btnLeft.style.backgroundColor = "#5fca5f"; 
+     btnRight.innerHTML = "RESET";  
+     btnRight.style.backgroundColor = "#908e8e"; 
+ }
```

Comment:
If you look into the styles file, you will see that the classes `.btn.start` and`.btn.stop` contain the background colors for the buttons.  So rather than setting the background color here, its better to toggle the classes.